drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenterTest.java

```diff
@@ -1641,4 +1655,17 @@ public void testRefreshView() {
         verify(parent).refreshScrollPosition();
         verify(layer).draw();
     }
+    
+    private void checkDTEventFired(int times){
```

Comment:
There are lots of events relating to the Decision Table; I'd prefer this to be renamed something more explicit like ```checkDecisionTableSelectionsChangedEventFired(int times)```.

Reply:
@manstis i agree as well.  originally thought of using generic argument type to allow for the multiple types but then decided until needed for this JUNit I';d keep it simple.  Will rename.